monitoring/grafana/dashboards/jsonnet/grafana_dashboards.jsonnet

```diff
@@ -289,8 +295,90 @@ local addStyle(alias, colorMode, colors, dateFormat, decimals, mappingType, patt
       RgwOverviewPanel(
         'Bandwidth by RGW Instance', 'Total bytes transferred in/out through get/put operations, by radosgw instance', 'bytes', 'short', 'sum by(rgw_host) (\n  (label_replace(rate(ceph_rgw_get_b[30s]), \"rgw_host\",\"$1\",\"ceph_daemon\",\"rgw.(.*)\")) + \n  (label_replace(rate(ceph_rgw_put_b[30s]), \"rgw_host\",\"$1\",\"ceph_daemon\",\"rgw.(.*)\"))\n)', '{{rgw_host}}', 8, 8, 7, 6),
       RgwOverviewPanel(
-        'PUT Latencies by RGW Instance', 'Latencies are shown stacked, without a yaxis to provide a visual indication of PUT latency imbalance across RGW hosts', 's', 'short', 'label_replace(rate(ceph_rgw_put_initial_lat_sum[30s]),\"rgw_host\",\"$1\",\"ceph_daemon\",\"rgw.(.*)\") / \nlabel_replace(rate(ceph_rgw_put_initial_lat_count[30s]),\"rgw_host\",\"$1\",\"ceph_daemon\",\"rgw.(.*)\")', '{{rgw_host}}', 15, 8, 6, 6)
-    ])
+        'PUT Latencies by RGW Instance', 'Latencies are shown stacked, without a yaxis to provide a visual indication of PUT latency imbalance across RGW hosts', 's', 'short', 'label_replace(rate(ceph_rgw_put_initial_lat_sum[30s]),\"rgw_host\",\"$1\",\"ceph_daemon\",\"rgw.(.*)\") / \nlabel_replace(rate(ceph_rgw_put_initial_lat_count[30s]),\"rgw_host\",\"$1\",\"ceph_daemon\",\"rgw.(.*)\")', '{{rgw_host}}', 15, 8, 6, 6),
+   
+      addRowSchema(false, true, 'RGW Overview - HAProxy Metrics') + {gridPos: {x: 0, y: 12, w: 9, h: 12}},
+      RgwOverviewPanel(
+        'Total responses by HTTP code', '', 'short', 'short', 'sum(irate(haproxy_frontend_http_responses_total{code=~\"$code\",instance=~\"$ingress_service\",proxy=~\"frontend\"}[5m])) by (code)', 'Frontend {{ code }}', 0, 12, 5, 12, '$datasource', true, true, true, true, true, true)
+        .addTargets(
+        [addTargetSchema('sum(irate(haproxy_backend_http_responses_total{code=~\"$code\",instance=~\"$ingress_service\",proxy=~\"backend\"}[5m])) by (code)', 1, 'time_series', 'Backend {{ code }}')])
+        .addSeriesOverride([
+          { "alias": "/.*Back.*/",
+            "transform": "negative-Y" },
+          {
+            "alias": "/.*1.*/",
+            "color": "#6ED0E0"
+          },
+          {
+            "alias": "/.*2.*/",
+            "color": "#7EB26D"
+          },
+          {
+            "alias": "/.*3.*/",
+            "color": "#1F78C1"
+          },
+          {
+            "alias": "/.*4.*/",
+            "color": "#CCA300"
+          },
+          {
+            "alias": "/.*5.*/",
+            "color": "#890F02"
+          },
+          {
+            "alias": "/.*other.*/",
+            "color": "#806EB7"
+          }
+        ]),
```

Comment:
Where do these color come from? Does it make sense to replace them with literals, so we use common colors across all our grafonnet stuff?

Reply:
I insist, we already have the following colors defined in our jsonnet (I'd prefer to replace those with literals):

`#3F2B5B`
`#6D1F62`
`#890F02`
`#BF1B00`
`#C15C17`
`#E0752D`
`#E5AC0E`
`#EAB839`
`#7EB26D`
`#508642`
`#3F6833`
`#447EBC`
`#0A50A1`

And here you're adding:
`#6ED0E0`
`#7EB26D`
`#1F78C1`
`#CCA300`
`#890F02`
`#806EB7`

I don't have any preference for any, but we should: a) avoid using literals and using meaningful names, b) reusing colors.

Reply:
Sprry I missed this, I think we don't need this colors to be defined here.